### ISSUE TITLE
URGENT :: ERROR ITMS-90725:  App Store needs iOS 13 SDK to build

### ISSUE THREAD

**Issue by manusis** [phonegap/build#666]
PRIOR TO CREATING AN ISSUE HERE, PLEASE POST TO THE FORUM:

https://forums.adobe.com/community/phonegap/build

Apple has stopped accepting builds that are not built with iOS 13 or later. 

ERROR ITMS-90725: "SDK Version Issue. This app was built with the iOS 12.1 SDK. As of June 30, 2020, all apps for iPhone or iPad must be built with the iOS 13 SDK or later, included with Xcode 11 or later."

**Comment by PegasusAwesome** [comment]
Have the exact same problem.
When will Adobe honor the contract they have with their paying customers and start supporting this?

**Comment by danzen** [comment]
Yes - this really sucks.  I do not appreciate the way Apple and Adobe deal with progress.  I guess we can pin our hopes on progressive web apps and just bypass the store all together.
